synchronizer/synchronizer.go

```diff
@@ -23,6 +23,12 @@ const (
 	// errStrUnknownBlock is the string returned by geth when querying an
 	// unknown block
 	errStrUnknownBlock = "unknown block"
+	// updateEthBlockNumThreshold is a threshold of number of ethereum blocks left to synchronize, such that
+	// if we have more blocks to sync than the defined value we can aggressively skip calling UpdateEth
+	updateEthBlockNumThreshold = 100
+	// While having more blocks to sync than updateEthBlockNumThreshold, UpdateEth will be called once in a
+	// defined number of blocks
+	updateEthFrequencyDivider = 100
```

Comment:
maybe it's better this value can be configurable in the config file by the owner

Reply:
I agree that these two parameters could be configurable